src/js/data/ui-update/UIMetadata.ts

```diff
@@ -13,3 +13,7 @@ export const UIMetadataSchema = `
     serverBuild: String
   }
 `;
+
+export const DEFAULT_UI_METADATA: UIMetadata = {
```

Comment:
should this file be in the plugin? It's only imported by files in plugins/*

Reply:
I originally had defined it in the plugin, but I felt like it made more sense being here with the interface definition.